src/forms/DepositForm.vue

```diff
@@ -150,27 +155,29 @@ const availableOptions = computed<FieldOption<AvailableOptionValue>[]>(() => [
     : []),
 ])
 
-const uid = uuidv4()
-
 const form = reactive({
-  available: availableOptions.value[0] || null,
+  availableOptionIdx: 0,
   amount: '',
 })
 
+const formAvailable = computed<FieldOption<AvailableOptionValue> | null>(
```

Comment:
looks like it should be renamed either with boolean prefix or with better description in name

Reply:
done, renamed: available -> balance